pkg/driver/controller.go

```diff
@@ -592,41 +578,35 @@ func (d *Driver) ControllerExpandVolume(ctx context.Context, req *csi.Controller
 	}
 
 	if splitVolumeId[0] == cloud.VolumePrefix {
-		v, err := d.cloud.DescribeVolume(ctx, volumeID)
-		if err != nil {
-			if err == cloud.ErrNotFound {
-				return nil, status.Errorf(codes.NotFound, "Volume not found with ID %q", volumeID)
-			}
-			return nil, status.Errorf(codes.Internal, "Could not get volume with ID %q: %v", volumeID, err)
-		}
+		return nil, status.Error(codes.Unimplemented, "Storage of volumeType Volume can not be scaled")
+	}
+	return nil, status.Errorf(codes.NotFound, "Volume not found with ID %q", volumeID)
+}
 
-		if newCapacity <= v.StorageCapacityQuotaGiB {
-			// Current capacity is sufficient to satisfy the request
-			klog.V(4).Infof("ControllerExpandVolume: current volume capacity of %d GiB matches or exceeds requested storage capacity of %d GiB, returning with success", v.StorageCapacityQuotaGiB, newCapacity)
-			return &csi.ControllerExpandVolumeResponse{
-				CapacityBytes:         util.GiBToBytes(v.StorageCapacityQuotaGiB),
-				NodeExpansionRequired: false,
-			}, nil
-		}
+func (d *Driver) ControllerGetVolume(ctx context.Context, req *csi.ControllerGetVolumeRequest) (*csi.ControllerGetVolumeResponse, error) {
+	return nil, status.Error(codes.Unimplemented, "")
+}
 
-		finalCapacity, err := d.cloud.ResizeVolume(ctx, volumeID, newCapacity)
-		if err != nil {
-			return nil, status.Errorf(codes.Internal, "resize failed: %v", err)
-		}
+func didUserDefineDriverProvidedParameters(userInput map[string]string) error {
+	driverDefinedParameters := []string{
```

Comment:
On first glance, storage capacity needs to be added to this list of user provided parameters